motr/m0crate/parser.c

```diff
@@ -55,6 +55,7 @@ enum config_key_val {
 	ADDB_INIT,
 	ADDB_SIZE,
 	LOG_LEVEL,
+	POOL_FID,
```

Comment:
Please note the comment on the next line.
I think the "POOL_FID" is workload-specific parameter, right?